apps/judicial-system/web/src/routes/Court/components/ReceptionAndAssignment/ReceptionAndAssignment.tsx

```diff
@@ -75,11 +75,11 @@ const ReceptionAndAssignment = () => {
     },
   )
 
-  const receiveCase = async (workingCase: Case, courtCaseNumber: string) => {
+  const receiveCase = async (workingCase: Case) => {
     if (workingCase.state === CaseState.SUBMITTED && !isTransitioningCase) {
       // Transition case from SUBMITTED to RECEIVED when courtCaseNumber is set
       const received = await transitionCase(
-        { ...workingCase, courtCaseNumber },
```

Comment:
This confuses me, I don't think this is needed. @Arnarkari93 @gudjong do you remember why we did this or is this just unneeded code?

Reply:
I am not sure. Could it be that the new `courtCaseNumber` is not updated for the `workingCase` state? Since `transitionCase` sends a mutation call to the server and updates the local state for `workingCase`

Reply:
That was what I thought but `transitionCase` does not do anything with the courtCaseNumber. I'mma say this is a mistake on our half